knife-windows.gemspec

```diff
@@ -15,7 +15,8 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version	= ">= 1.9.1"
   s.add_dependency "winrm", "~> 1.3"
-  s.add_dependency "winrm-s", "~> 0.2"
+  s.add_dependency "winrm-s", "~> 0.3.0.dev.0"
```

Comment:
maybe we should edit ```lib\knife-windows\version.rb``` to something like this:

```
    VERSION = "1.0.0.dev.0" # FIXME: release winrm-s 0.3.0 and update gemspec before release
```

We probably don't need to, and it's a hack. Maybe just typing this will make me remember :)

I'll make a card on our board.